server/views/icons/a11y/stt.svg

```diff
@@ -0,0 +1 @@
+<svg id="Layer_1" viewBox="0 0 26 26"><style>.st0{fill:#231f20}</style><path class="st0" d="M23.9 11.4H22v5h-1.4v-5h-1.8v-1.1h5.1v1.1zm-7.4 1.8l2.1 3.2H17l-1.3-2.1-1.4 2.1h-1.5l2.2-3.2-2-2.9h1.6l1.2 2 1.3-2h1.5l-2.1 2.9zm-4.1 3.2H7.7v-6.2h4.6v1.1H9.1v1.3h3v1.1h-3v1.5h3.3v1.2zm-5.2-5H5.3v5H4v-5H2.1v-1.1h5.1v1.1zm18.8 11V3.6c0-2-1.6-3.6-3.6-3.6H3.6C1.6 0 0 1.6 0 3.6v18.8c0 2 1.6 3.6 3.6 3.6h18.8c2 0 3.6-1.6 3.6-3.6"/></svg>
```

Comment:
Could this be tidied up in line with the other svgs?

```
<svg viewBox="0 0 26 26"><path fill="#1d1d1d" class="icon__shape" d="M23.9 11.4H22v5h-1.4v-5h-1.8v-1.1h5.1v1.1zm-7.4 1.8l2.1 3.2H17l-1.3-2.1-1.4 2.1h-1.5l2.2-3.2-2-2.9h1.6l1.2 2 1.3-2h1.5l-2.1 2.9zm-4.1 3.2H7.7v-6.2h4.6v1.1H9.1v1.3h3v1.1h-3v1.5h3.3v1.2zm-5.2-5H5.3v5H4v-5H2.1v-1.1h5.1v1.1zm18.8 11V3.6c0-2-1.6-3.6-3.6-3.6H3.6C1.6 0 0 1.6 0 3.6v18.8c0 2 1.6 3.6 3.6 3.6h18.8c2 0 3.6-1.6 3.6-3.6"/></svg>
```

Reply:
Done